llama-index-integrations/embeddings/llama-index-embeddings-text-embeddings-inference/pyproject.toml

```diff
@@ -27,12 +27,11 @@ exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-embeddings-text-embeddings-inference"
 readme = "README.md"
-version = "0.1.3"
+version = "0.1.4"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.1"
-llama-index-embeddings-huggingface = ">=0.1.1"
```

Comment:
Actually, all these deps are in the `llama-index-utils-huggingface` package 👁️ we can just add that as a dep and update imports, instead of modifying core

Reply:
I pushed that :)